### PR TITLE
build: update files based on check warnings and errors.

### PR DESCRIPTION
**Pull request by lwjohnst86**
One good habit to get into is to run checks before pushing. You run a check with `Ctrl-Shift-E` while in RStudio or `devtools::check()` in the Console. This does basic checks that would fulfill the requirements of CRAN. This PR is fixing most of these issues that have shown up. Review the commit history to see why I've changed each thing.

### CODE REVIEW

File: inst/WORDLIST [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 CMD
-DuckDB
-Stata
+HbA
+HbA1c
+Isaksen's
+Lifecycle
+Metformin
+ORCID
+OSCD
```
Comment (Aastedet) [inline inst/WORDLIST:9]: It's OSDC

It's too bad that one algorithm ends with DC while the other (RSCD) ends with CD :)

**Review by Aastedet — CHANGES_REQUESTED** [review]
Just the OSCD typo

File: inst/WORDLIST (hunk 1 above)
Comment (lwjohnst86) [inline inst/WORDLIST:9]:
```suggestion
OSDC
```

**Review by signekb — APPROVED** [review]
👍

**Comment by lwjohnst86** [comment]
@Aastedet they are in the Suggests section of the DESCRIPTION file. To install all dependencies, you can use the `pak::pak()` function to install even Suggests.

**Pull request merged**